L1Trigger/L1CaloTrigger/interface/Phase2L1CaloEGammaUtils.h

```diff
@@ -928,19 +922,23 @@ namespace p2eg {
     return is_iso;
   }
 
-  inline bool passes_ss(float pt, float ss) {
+  inline bool passes_ss(float pt, float eta, float ss) {
     bool is_ss;
     if (pt > 130)
       is_ss = true;
+    else if ((abs(eta) > 0.4 && abs(eta) < 0.5) || (abs(eta) > 0.75 && abs(eta) < 0.85))  // temporary adjustment
+      is_ss = ((loose_ss_offset + c1_ss * std::exp(-c2_ss * pt)) <= ss);
```

Comment:
This part contains many magic numbers, including pt cut above. Is it possible to define them as `constexpr float` before?

Reply:
@srimanob done